boards/redboard_redv/README.md

```diff
@@ -0,0 +1,62 @@
+Redboard Red-V B RISC-V Board
+==================================
+
+- https://www.sparkfun.com/products/15594
+
+Arduino-compatible dev board for RISC-V clone of the Hifive1.
+
+Programming
+-----------
+
+Running `make flash` should load the kernel onto the board. You will need a
+relatively new (i.e. from git) version of OpenOCD.
+
+The kernel also assumes there is the default HiFive1 software bootloader running
+on the chip.
+
+Running in QEMU
+---------------
+
+The HiFive1 application can be run in the QEMU emulation platform for RISC-V, allowing quick and easy testing.
+
+Unfortunately you need QEMU 5.1, which at the time of writing is unlikely to be avaliable in your distro. Luckily Tock can build QEMU for you. From the top level of the Tock source just run `make ci-setup-qemu` and follow the steps.
```

Comment:
We also probably don't need this sentence any more, or at least can re-word it as 5.1 is getting a little old